style.css

```diff
@@ -0,0 +1,733 @@
+/* stylelint-disable no-descending-specificity */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #f7f7f9;
+}
+
+header {
+  font-family: 'Poppins', sans-serif;
+  display: flex;
+  justify-content: space-between;
+  padding: 1em 2em;
+  float: left;
+  position: fixed;
+  top: 10px;
+  width: 100%;
+  z-index: 100;
```

Comment:
Kindly consider adding background color to the header to prevent conflict between navbar and other sections while scrolling down 😁  

![Screenshot 2023-11-25 115646](https://github.com/HsuMyatPhoo/PortfolioFinishMobileVersion/assets/102011195/ad51fb95-6dd6-4106-9891-b99f0825c347)